File: JavaScript/Manzano/Pagina74a.js
// Programa que lê 10 elementos de uma matriz tipo vetor e os apresenta no final.

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const vet = [];

function lerElementoVetor() {
  rl.question("Adicione um número ao vetor: ", function (input) {
    const numero = parseInt(input);
    vet.push(numero);

    if (vet.length < 10) {
      lerElementoVetor();
    } else {
      rl.close();
      apresentarVetor();
    }
  });
}

function apresentarVetor() {
  for (let i = 0; i < vet.length; i++) {
    process.stdout.write(vet[i] + " ");
  }
}

lerElementoVetor();

File: Java/src/Manzano/Pagina74a.java
package Manzano;
//Program que lê 10 elementos de uma matriz tipo vetor e os apresenta  no final.

import java.util.Scanner;

public class Pagina74a {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        int[] vet = new int[10];

        for (int i = 0; i < 10; i++) {
            System.out.println("Adicione um número ao vetor: ");
            vet[i] = scanner.nextInt();
        }
        scanner.close();

        for (int i = 0; i < 10; i++) {
            System.out.print(vet[i] + " ");
        }

    }
}
